notebooks/MAST/spectraldb_and_specviz/spectraldb_and_specviz.ipynb

```diff
@@ -0,0 +1,919 @@
+{
```

Comment:
<p>I think it's ok to have the search be somewhat arbitrary. However, if you want something specific to search for, for the spectrum you visualize at the end of this tutorial, the big emission line at ~16 microns is a NeIII (neon III) feature, the line around ~13 microns is a NeII (neon II) feature, and the emission lines around ~12 microns and ~17.5 microns are H2 (molecular hydrogen) lines. You could design your search around one of those lines if you wanted.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I decided to keep it arbitrary.</p>